website2/website/scripts/replace.js

```diff
@@ -30,14 +29,37 @@ replace(options)
     });
 }
 
+
 const versions = getVersions();
 
 const latestVersion = versions[0];
 
+const bazelVersions = {
+    '0.20.0': '0.14.1',
+    '0.20.1': '0.26',
+    '0.20.2': '0.26',
```

Comment:
Suggest removing 0.20.2.  Or maybe you mean "latest"?

Reply:
Okay!.